src/main/scala/io/scala/Page.scala

```diff
@@ -33,13 +33,13 @@ case class IndexPage(withDraft: Option[Boolean] = None, conference: Option[Strin
     with Draftable
     with Routeable:
   def title: String = "Home"
-case class TalksPage(withDraft: Option[Boolean] = None, conference: Option[String] = None)
+case class SessionsPage(withDraft: Option[Boolean] = None, conference: Option[String] = None)
```

Comment:
Add a `kind` attribute (or any name you'd prefer) with this type

```scala
enum Kind
  case Talk, Workshop
```

Then add a new route with the same schema as the talks (now sessions) route but with `/workshop` as last path fragment